### ISSUE TITLE
Turning this into a CLI app

### ISSUE THREAD

**Issue by BolajiOlajide** [rm-hull/sql_graphviz#17]
Hey!

Really awesome tool out there. I was wondering if it's possible to package this into an executable that can be run from the command line.
One can make use [typer](https://github.com/tiangolo/typer).

I'm happy to work on this if you think it's a good thing to have. cc @rm-hull

**Comment by rm-hull** [comment]
Yes, sounds like a good idea. We should also think about publishing to pypi too (although that should be a separate PR)
